test/testdrive/mzcompose.py

```diff
@@ -91,6 +91,11 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
     materialized = Materialized(
         default_size=args.default_size,
+        environment_extra=[
+            "MZ_AWS_ACCOUNT_ID=123456789000",
+            "MZ_AWS_EXTERNAL_CONNECTION_ROLE=arn:aws:iam::123456789000:role/MaterializeConnection",
+            "MZ_AWS_EXTERNAL_ID_PREFIX=eb5cb59b-e2fe-41f3-87ca-d2176a495345",
+        ],
```

Comment:
We try to keep testdrive runnable without mzcompose—i.e., via `cargo run --bin testdrive`. This will break that ability, as aws-accounts.td will depend on these variables being set. 

It would probably be best to move the AWS connection tests to a new mzcompose composition that starts localstack. Look at test/secrets-aws-secrets-manager for inspiration. @MaterializeInc/testing can help. Possibly the right thing to do is to reuse the existing composition in test/secrets-aws-secrets-manager and simply add a new workflow to the file.

Reply:
I will move it to test/secrets-aws-secrets-manager. I was not sure whether to create a completely new mzcompose or use something existing.

Reply:
done